packages/x-components/src/composables/use-x-bus.ts

```diff
@@ -23,7 +24,12 @@ export function useXBus(): UseXBusAPI {
   if (currentComponent && currentXComponent) {
     currentComponent.xComponent = currentXComponent;
   }
-
+  let bus: XBus<XEventsTypes, WireMetadata>;
+  try {
```

Comment:
Why is it a `try/catch` and not an `undefined` checking? 🤔

Reply:
The `XPlugin` getter for the bus relies on the [getInstance](https://github.com/empathyco/x/blob/main/packages/x-components/src/plugins/x-plugin.ts#L75) static method that throws if the plugin was not instantiated.

Reply:
Oouh! Makes sense. Could we add a small comment here with that reason? Or with a `@remarks` tag in the composable doc